src/governance.cpp

```diff
@@ -265,7 +266,6 @@ void CGovernanceManager::CheckOrphanVotes(CGovernanceObject& govobj, CGovernance
             fRemove = true;
         }
         else if(govobj.ProcessVote(NULL, vote, exception)) {
```

Comment:
I don't really want to be making cosmetic changes, or other unrelated changes, until the syncing and forking issues are resolved.